totalRP3_Extended_Tools/Script/Effects/Effects.lua

```diff
@@ -77,11 +77,36 @@ local function text_init()
 	end
 end
 
+local MAX_CHARACTERS_IN_MACRO = 255;
+
+local function checkCharactersLimit()
+	local macro = TRP3_EffectEditorMacro.MacroText.scroll.text:GetText();
+	local numberOfCharactersInMacro = strlen(macro);
+
+
+	-- Always start with confirm enabled, then check for disable.
+	TRP3_EffectEditorMacro.confirm:Enable();
+	local color = WHITE_FONT_COLOR;
+	if numberOfCharactersInMacro > MAX_CHARACTERS_IN_MACRO then
+		color = RED_FONT_COLOR;
+		-- Too many characters, disable confirm.
+		TRP3_EffectEditorMacro.confirm:Disable();
+	end
+
+	-- Always disable confirm if the message text is empty
+	if strlen(macro) == 0 then
+		TRP3_EffectEditorMacro.confirm:Disable();
+	end
+
+	TRP3_EffectEditorMacro.CharactersCounter:SetTextColor(color:GetRGB())
+	TRP3_EffectEditorMacro.CharactersCounter:SetText(strconcat(MAX_CHARACTERS_IN_MACRO - numberOfCharactersInMacro));
```

Comment:
```suggestion
	TRP3_EffectEditorMacro.CharactersCounter:SetText(MAX_CHARACTERS_IN_MACRO - numberOfCharactersInMacro);
```
Concatenation of a single element is unnecessary.